guides/common/modules/proc_generating-a-list-of-installed-packages.adoc

```diff
@@ -0,0 +1,18 @@
+[id="Generating_a_List_of_Installed_Packages_{context}"]
+= Generating a List of Installed Packages
+
+Use this procedure to generate a list of installed packages in *Report Templates*.
+
+.Procedure
+
+. In the {ProjectWebUI}, navigate to *Monitor* > *Report Templates*.
+. Select the *Host - All Installed Packages* report template.
+There is a host filter, so it does not have to display every host you have available in the report.
+. After you verify the report template, return to the *Report Templates* screen.
```

Comment:
Without the removed sentence, I think these two steps are also unnecessary as they lack context and are not required to complete the procedure. Removing them seems like a good idea.